### ISSUE TITLE
Exception Processing Message 0xc0000005 - Unexpected Parameters Error

### ISSUE THREAD

**Issue by john7200** [vawser/Smithbox#53]
Greetings.

I had constant errors with "Exception Processing Message 0xc0000005 - Unexpected Parameters" when starting Smithbox v1.0.15.1 (Low Requirement version).

There is no log generated or produced by Smithbox since it fails to start so here is the stack log/trace:

> at ImGuiNET.ImGuiNative.igNewFrame()
> at ImGuiNET.ImGuiNative.igNewFrame()
> at ImGuiNET.ImGui.NewFrame()
> at StudioCore.Graphics.OpenGLImGuiRenderer.OnSetupDone()
> at StudioCore.Smithbox..ctor(StudioCore.Graphics.IGraphicsContext, System.String)
> at Smithbox_LowRequirements.Program.Main(System.String[])

I used Smithbox v1.0.15 before and I have checked both v1.0.15 and v1.0.15.1 recently and found out the error only occurred with v1.0.15.1 at startup with Smithbox failed to start everytime, while v1.0.15 didn't have this error. I used the Low Requirement on both versions.

I'm using Windows 10 Pro 22H2 and haven't installed or updated anything in the last 2 weeks, with the exception being NVIDIA GPU driver (I'm using NVIDIA's developer driver).

Thanks in advance for any help or assistance.

**Comment by john7200** [comment]
Greetings.

I have tried Smithbox v1.0.16.1 and it doesn't crash and trigger the error "Exception Processing Message" like v1.0.16 or v1.0.15.1 but it produced crash logs about crash at startup related to cimgui.dll.

Here is one of the crash logs (there are multiple crash logs but all are the same since I tried to run Smithbox a few times):

[Log 2024-7-19--10-24-39.txt](https://github.com/user-attachments/files/16300861/Log.2024-7-19--10-24-39.txt)

I may be mistaken but it seems like both the .exe from Smithbox and Smithbox_LowRequirements folder are the same? Both .exe uses the same filename "Smithbox.exe" and have the same SHA1 hash.
